src/configure/templateInputHelper/InputControlProvider.ts

```diff
@@ -10,24 +10,41 @@ import { VisibilityHelper } from "./utilities/VisibilityHelper";
 export class InputControlProvider {
     private _pipelineTemplate: ExtendedPipelineTemplate;
     private _inputControlsMap: Map<string, InputControl>;
+    private _context: { [key: string]: any };
 
     constructor(pipelineTemplate: ExtendedPipelineTemplate, context: { [key: string]: any }) {
         this._pipelineTemplate = pipelineTemplate;
         this._inputControlsMap = new Map<string, InputControl>();
+        this._context = context;
         this._createControls(context);
     }
 
-    public async getAllPipelineTemplateInputs(azureSession: AzureSession, resourceNode?: GenericResource) {
+    public async overrideParameters(inputControl: InputControl) {
```

Comment:
this function is not async remove it. Add void return type.

Reply:
updated